cpp/build/support.mk

```diff
@@ -96,9 +107,15 @@ else
 pkgconfigdir ?= $(shell pkg-config --variable pc_path pkg-config | awk '{split($$0,a,":"); print a[1]}')
 endif
 
+ifeq ($(BITBAKE_ENV),1)
+sysconfdir := $(PREFIX)/etc/openzwave/
+includedir := $(PREFIX)/include/openzwave/
+docdir := $(PREFIX)/share/doc/openzwave-$(VERSION).$(VERSION_REV)
+else
 sysconfdir ?= $(PREFIX)/etc/openzwave/
```

Comment:
why can't PREFIX be specified during the first run?

You do realise that PREFIX is used as part of the source compilation, so if its not consistent, then things could break?

(for instance, we use PREFIX when compiling to specify the default location to search for the device database...)